cmd/monitoring/main.go

```diff
@@ -2,20 +2,30 @@ package main
 
 import (
 	"context"
+	"fmt"
+	"log"
 
 	"github.com/gagliardetto/solana-go/rpc"
 	relayMonitoring "github.com/smartcontractkit/chainlink-relay/pkg/monitoring"
-	"github.com/smartcontractkit/chainlink-solana/pkg/monitoring"
 	"github.com/smartcontractkit/chainlink/core/logger"
+
+	"github.com/smartcontractkit/chainlink-solana/pkg/monitoring"
 )
 
 func main() {
-	coreLog := logger.NewLogger()
-	log := logWrapper{coreLog}
+	coreLog, closeLggr := logger.NewLogger()
+	defer func() {
+		if closeLggr != nil {
+			if err := closeLggr(); err != nil {
+				log.Println(fmt.Sprintf("Error while closing Logger: %v", err))
+			}
+		}
```

Comment:
I think this nil check is still unnecessary, no?
```suggestion
		if err := closeLggr(); err != nil {
			log.Println(fmt.Sprintf("Error while closing Logger: %v", err))
		}
```

Reply:
Indeed this was odd check, removed.